src/editor.js

```diff
@@ -2889,13 +2889,19 @@ config.defineOptions(Editor.prototype, "editor", {
 
             var gutterKeyboardHandler;
 
-            // Prevent focus to be captured when tabbing through the page. When focus is set to the content div, 
-            // press Enter key to give focus to Ace and press Esc to again allow to tab through the page.
+            // If keyboard a11y mode is enabled we:
+            // - Enable keyboard operability gutter.
+            // - Prevent tab-trapping.
+            // - Hide irrelevant elements from assistive technology.
+            // - On Windows, set more lines to the textarea.
             if (value){
                 this.renderer.enableKeyboardAccessibility = true;
                 this.renderer.keyboardFocusClassName = "ace_keyboard-focus";
 
                 this.textInput.getElement().setAttribute("tabindex", -1);
+                // VoiceOver on Mac OS works best with single line in the textarea, the screen readers on
+                // Windows work best with multiple lines in the textarea.
+                this.textInput.setNumberOfExtraLines(useragent.isWin ? 3 : 0);
```

Comment:
what about linux?

Reply:
Good question, I don't have quick access to a Linux desktop so I couldn't test this so I didn't want to change anything for Linux. The most used screen reader on Linux seems to be [orca](https://help.gnome.org/users/orca/stable/index.html.en).